_posts/2020-02-21-antifragile.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "AntifrAGILILITY"
+date:   2020-02-21 19:00:00 -0300
+categories: agile
+---
+
+Recently, I gave a talk on [agile meetup][agile meetup] about something that's been bothering me for a while. This is a post-talk write-up that I intend to expand further.
+
+I've had mixed feeling about agile methodologies for a while and definitely a problem with the agile cargo culting.
+I was never quite certain why since my day-to-day job is mostly technical and I had the luxury of not having to care.
+Until recently, for 2 years, I was in a semi-leadership position where I had to care.
+Upon reflecting on that experience and reading [antifragile][antifragile], I started making connections everywhere.
+My confirmation bias was making me think everything could be explained through this concept, and "agility" wasn't an exception.
+
+To gloss over a bit on the concept, `antifragility` is a recently coined term by [Nassim Nicholas Taleb][taleb] that describes a property of systems that gain from disorder, in contrast to the fragile that loses from disorder and the robust that doesn't care much.
+Complex systems such as life, communities and ecossystems can't be understood in full, but we can measure it's fragility by an assymetry test: if a system gains more from a stressor than it loses, it's antifragile.
+
+Agile methodologies (or now the bastardized Agile), upon observing from this concept, are methodologies that took advantage of this property.
+
+If you think about software as a system encompassing all stakeholders (from code and machine to clients and investors), it's a system with multiple dependencies, interactions and very complex.
+You cannot predict the outcome feasibly [as we intuitively feel sometimes][no estimates], but, drawing from the concepts of antifragility, you can measure some of its properties.
+
+The waterfall method was very risky and fragilized the team/company since the project could be all wrong in the end of a cycle and invalidate all the work done, so the agile methodologies reduced the risk in smaller ones through iterative cycles and feedback.
+It introduced stressors to the systems such as recurring requirement changes - and encouraged the team to embrace it - and continuous feedback from client through collaboration over contract negotiation.
+It also recognized that, by letting individuals experiment and fail, it created a loop of constant trial, tinkering and error.
+
+A lot of the practices may make the system antifragile.
+Pair programming shares knowledge and creates redundancy (a push from fragile to robust), increasing the [bus factor][bus factor].
+The simplicity principle of reducing unnecessary work also contributes to a more predictable change in the system [via negativa][via negativa].
+It also gives autonomy to teams, reducing cascading errors that the entire organization suffers.
```

Comment:
s/that the entire organization suffers/that makes the entire organization suffer ? or something like this?